### ISSUE TITLE
Failed to open any project after eddy plugin installation

### ISSUE THREAD

**Issue by duponter** [eddysystems/issues#2]
Hi,

As your plugin sounds promising, I decided to give it a try.

I installed version 0.1-14 (the one updated on Feb 23) via the regular channel Plugins > Browse Repositories.
After the restart, I want to open 1 of my projects, but it fails with this message:
_Cannot load project: com.intellij.ide.plugins.PluginManager$StartupAbortedException: com.intellij.diagnostic.PluginException: LVTT entry for 'request' in class file com/amazonaws/auth/AbstractAWSSigner does not match any LVT entry [Plugin: com.eddy.generator]_

See also this screenshot:
![screenshot from 2015-02-24 09 26 16](https://cloud.githubusercontent.com/assets/205130/6345516/42bb9fb8-bc07-11e4-885d-980e0bca7bf4.png)

Choosing another project or re-installing the plugin didn't change a thing, only deactivating does...

I am using IntelliJ Ultimate Edition 14.0.3 Build: 139.1117.

Kind regards,
Erwin Dupont

**Comment by martinwicke** [comment]
I'm looking at this right now. There seems to be a hidden dependency that escaped us. We should have an update shortly. Thank you!

**Comment by martinwicke** [comment]
I tried installing eddy on a fresh computer with a fresh install of IntelliJ 14, 139-1117. Sadly, it worked, so something more subtle is going on. I have some questions

- You write about a message "... LVTT entry for 'request' ..."

**Comment by martinwicke** [comment]
I tried installing eddy on a fresh computer with a fresh install of IntelliJ 14, 139-1117. Sadly, it worked, so something more subtle is going on. I have some questions:
- You write about a message "... LVTT entry for 'request' ...", but the screenshot shows another message. Is there an order to these? Did one happen first?
- Do you have the scala plugin installed?

Thank you.

**Comment by martinwicke** [comment]
Because I cannot reproduce the bug, could you try installing this version:

https://eddy.systems/download/eddy-0.2pre-14.jar

It probably fixes the "LVTT" problem, and I suspect the other message is related.

**Comment by duponter** [comment]
First of all, let me clarify the mismatch between the text message and the screenshot. I had already composed this issue report based on the 0.1-14 of February, 20th. Before submitting the report, I saw there was another 0.1-14 plugin waiting for approval on February, 23rd. Thus, I waited for this version to be released and retried installing the plugin. I still got an error on project startup, but I hadn't noticed the message was different.

The 'LVTT' problem is indeed the message I get:
![eddy-1-14](https://cloud.githubusercontent.com/assets/205130/6367347/46f7068c-bcd2-11e4-94e1-8544cb62e4f6.png)

Now the good news: I have installed version 0.2pre-14 and my projects start up normally! 

Finally, this may not be relevant anymore, but it can nonetheless be useful information for the future:
- I do not have the Scala plugin installed.
- I am running IntelliJ on Ubuntu Linux 14.04.

Thanks for the quick replies...
